.travis.yml

```diff
@@ -18,13 +18,17 @@ install:
   # Install dependencies
   - conda create -n test-environment python=$TRAVIS_PYTHON_VERSION
   - source activate test-environment
-  - conda install pytest numpy pip coverage toolz pandas cython scikit-learn cytoolz dill networkx pyzmq psutil
+  - conda install pytest numpy pip coverage toolz pandas cython scikit-learn cytoolz dill networkx pyzmq psutil ipython
   - pip install chest
   - pip install git+https://github.com/blosc/bcolz --upgrade
 
   # Install dask
   - python setup.py install
 
+  # start the ipcluster, it takes a while to start
+  - ipcluster start -n 4 &
+  - sleep 20
```

Comment:
Looks like `ClusterTestCase` might be worth looking in to.  This feels a bit cleaner because we're using python to call python code and don't have the sleep.